package.json

```diff
@@ -36,10 +36,7 @@
   "devDependencies": {
     "jasmine-node": "^1.14.5",
     "mongodb": "^1.4.30",
-    "mongoose": "3.8.x",
+    "mongoose": "^3.8.x",
```

Comment:
The dependency you are referring to is a dev dependency, so i think no issue here ;)